internal/cache/bbolt/bbolt.go

```
@@ -58,182 +58,109 @@ func (c *Cache) Connect() error {
 		return err
 	}
 
-	go c.Reap()
+	// Load Index here and pass bytes as param2
+	indexData, _ := c.retrieve(cache.IndexKey, false)
+	c.Index = cache.NewIndex(c.Name, c.Config.Type, indexData, c.Config.Index, c.BulkRemove, c.storeNoIndex)
 	return nil
 }
 
 // Store places an object in the cache using the specified key and ttl
 func (c *Cache) Store(cacheKey string, data []byte, ttl int64) error {
+	return c.store(cacheKey, data, ttl, true)
+}
 
-	expKey, dataKey := c.getKeyNames(cacheKey)
-	expiration := []byte(strconv.FormatInt(time.Now().Unix()+ttl, 10))
+func (c *Cache) storeNoIndex(cacheKey string, data []byte) {
+	c.store(cacheKey, data, 31536000, false)
```

Ignore the error here?

---

updated to handle the error. thanks!